Scripts/Runtime/Entities/TaskSystem/TaskStream/TaskStream.cs

```diff
@@ -7,30 +9,88 @@ namespace Anvil.Unity.DOTS.Entities.Tasks
     /// <see cref="AbstractTaskSystem"/>.
     /// </summary>
     /// <typeparam name="TInstance">The type of <see cref="IEntityProxyInstance"/> in this stream</typeparam>
+    // ReSharper disable once ClassNeverInstantiated.Global
     public class TaskStream<TInstance> : AbstractTaskStream
         where TInstance : unmanaged, IEntityProxyInstance
     {
+        private readonly string m_TypeString;
```

Comment:
Same as before

m_TypeName?
Worth caching like this?

Reply:
Removed